vertx-web/src/main/java/io/vertx/ext/web/handler/impl/HttpStatusException.java

```diff
@@ -0,0 +1,27 @@
+package io.vertx.ext.web.handler.impl;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+
+public class HttpStatusException extends Throwable {
```

Comment:
Do we need this to extend Throwable? Is it just for avoiding the stacktrace? In this case we could use `java.lang.Exception#Exception(java.lang.String, java.lang.Throwable, boolean, boolean)`